package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "warrioz-frontend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server --mode development"
```

Comment:
@kabundege can you add script that can build the project and serve the build version on production and a Dev script.